packages/api/src/beacon/routes/beacon/block.ts

```diff
@@ -236,7 +247,11 @@ export type Api = {
     indices?: number[]
   ): Promise<
     ApiClientResponse<{
-      [HttpStatusCode.OK]: {executionOptimistic: ExecutionOptimistic; data: deneb.BlobSidecars};
+      [HttpStatusCode.OK]: {
+        data: deneb.BlobSidecars;
+        executionOptimistic: ExecutionOptimistic;
+        finalized: Finalized;
```

Comment:
Hmmm I don't think we need `executionOptimistic` or `finalized` for this endpoint: https://ethereum.github.io/beacon-APIs/#/Beacon/getBlobSidecars

Reply:
Yeah it does not, but we already had execution_optimistic field here and I think it should?

I have opened an issue for this a few days ago on the spec
- https://github.com/ethereum/beacon-APIs/issues/439

Reply:
As this API is not used by other clients but only users that want to retrieve chain data I would prefer to keep the metadata, unless someone has a good argument why it shouldn't be there

Reply:
yeah I think it's better to have it as it's still compliant to the spec